File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/03.Conditional-Statements-Advanced/02-exercise/07.hotelRoom.js
function hotelRoom(input) {

    let mounth = input[0];
    let nights = Number(input[1]);

    let priceForStudio = 0;
    let priceForApartment = 0;

    if (mounth === "May" || mounth === "October") {
        priceForStudio = nights * 50;
        priceForApartment = nights * 65;
        if (nights > 7 && nights < 14) {
            priceForStudio = priceForStudio * 0.95;
        } else if (nights > 14) {
            priceForStudio = priceForStudio * 0.7;
        }
    } else if (mounth === "June" || mounth === "September") {
        priceForStudio = nights * 75.2;
        priceForApartment = nights * 68.7;
        if (nights > 14) {
            priceForStudio = priceForStudio * 0.8;
        }
    } else if (mounth === "July" || mounth === "August") {
        priceForStudio = nights * 76;
        priceForApartment = nights * 77
    }

    if (nights > 14) {
        priceForApartment = priceForApartment * 0.9;
    }

    console.log(`Apartment: ${priceForApartment.toFixed(2)} lv.`);
    console.log(`Studio: ${priceForStudio.toFixed(2)} lv.`);
}
hotelRoom(["August", "20"]);

File: Python_Basics/Homeworks-And-Labs/03.conditional_statements_advanced/02.exercise/07.hotel_room.py
month = input()
nights = int(input())

priceForStudio = 0
priceForApartment = 0

if month == "May" or month == "October":
    priceForStudio = nights * 50
    priceForApartment = nights * 65
    if nights > 7 and nights < 14:
        priceForStudio = priceForStudio * 0.95
    elif nights > 14:
        priceForStudio = priceForStudio * 0.7
elif month == "June" or month == "September":
    priceForStudio = nights * 75.2
    priceForApartment = nights * 68.7
    if nights > 14:
        priceForStudio = priceForStudio * 0.8
elif month == "July" or month == "August":
    priceForStudio = nights * 76
    priceForApartment = nights * 77

if nights > 14:
    priceForApartment = priceForApartment * 0.9

print(f'Apartment: {priceForApartment:.2f} lv.')
print(f'Studio: {priceForStudio:.2f} lv.')
